composer.json

```
@@ -32,7 +32,7 @@
     "require": {
         "php": ">=7.4 <8.4",
         "benmorel/weakmap-polyfill": "^0.4.0",
-        "psr/log": "^1.0",
+        "psr/log": "^3.0",
```

```suggestion
        "psr/log": "^1.0 || ^2.0 || ^3.0",
```

we cannot limit the minimal version

---

Agreed.